content/departments/cloud/index.md

```diff
@@ -290,3 +290,21 @@ For #ce teammates, please reach out to #cloud and include link to this FAQ
 
 Cloud supports all code-hosts types (self-managed and Cloud-managed), but it currently requires the code-host to have a public IP.
 More context [here](https://docs.google.com/document/d/14S3jn0bV03WdeT1H36omvtGJFoIFJjM-3ZA1qIyIl7o/edit)
+
+### FAQ: How do I figure out the GCP Project ID for a customer?
+
+The best way to determine the project ID for a given customer is to look up the customer in the `deploy-sourcegraph-managed` repo using the following query on S2:
+
+```
+repo:^github\.com/sourcegraph/deploy-sourcegraph-managed$ file:config\.yaml lang:yaml customer: :[_\n]
```

Comment:
@danieldides sorry that so late I review it, but this won't work for PLG automated trials - it should be grep by emailDomain, b/c customer names are random 🙂  but the second option with gcloud and `domain` field works for all 👍

Reply:
Great docs btw 👏